### ISSUE TITLE
Trakttv Scrobbler Not Showing Up in Channels

### ISSUE THREAD

**Issue by brianmichaud** [trakt/Plex-Trakt-Scrobbler#381]
Hello, 

I can't seem to find out how to get this plug-in to display on PMS. I have it loaded in the proper directories (/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins) but it just won't show up. I'd add a log file, but I don't seem to be generating one. . . 

I can't get access to the WebTools portal either. . . not sure what is going on. Any help would be appreciated. 

Thanks.

**Comment by fuzeman** [comment]
Might be an issue with permissions on the plugin directory, did you try the [permissions fix](https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Installation%20-%20Debian%20%28Terminal%29#fix-permissions) listed on the installation guide?

If the permissions look correct on the plugin directories, posting the "Plex Media Server.log" file here will help me identify why the plugins aren't starting up correctly.

**Comment by brianmichaud** [comment]
Yeah, that easily could be. I tried reinstalling the scrobbler manually, but for some reason the permissions on the Trakttv.bundle folder are not the same as other Plug-ins in the folder. 

I tried the permissions fix twice and can't get it to change either. . . 

Here is a link to the Plex Media Server.log file. . . 

[https://www.odrive.com/s/e60720ee-0be8-4337-b5a4-adb05d9bdca2-58584b7b](https://www.odrive.com/s/e60720ee-0be8-4337-b5a4-adb05d9bdca2-58584b7b)

Any other help would be most appreciated. 

Thank you.

**Comment by fuzeman** [comment]
Found this in your log file:
```
Warning: ex: boost::filesystem::last_write_time: No such file or directory: "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins", couldn't check file: /var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins
```

Which is possibly related to this issue: https://forums.plex.tv/discussion/229998/is-plex-media-server-no-longer-supported-on-the-netgear-readynas-212

--------------

Which ReadyNAS device do you have? it sounds like this issue might be related to not using the correct build of Plex Media Server on ReadyNAS 2xx devices (should be using `ARM 6.x RN2xx` instead of `ARM 6.x` on the download page)..

I would try updating Plex Media Server *(you are a few releases behind)*, and ensure you use the correct build of Plex Media Server for your device.

**Comment by brianmichaud** [comment]
Hmmm. . . very strange. That folder definitely is there. 

I am using PMS on a Raspberry Pi 3, so there isn't any ReadyNAS involved. Although due to the ARM architecture I think the RPi build of PMS is based on the ReadyNAS version.

I'll update PMS again and see if that somehow fixes it. 

Anything else, please feel free to let me know. 

Thanks.

**Comment by brianmichaud** [comment]
Unfortunately I haven't come across any updates to PMS on Raspberry Pi that have fixed this issue.

**Comment by fuzeman** [comment]
> Can you confirm that these are the proper permissions for the Trakttv.bundle directory?

That looks correct, but the permissions need to be applied recursively (unable to confirm that from the screenshot).

-------------

I just tested the plugin again on my Raspberry Pi 3 and couldn't reproduce the issue you are encountering.

Tested with:
 - **Raspbian:** 2016-03-18 (jessie)
 - **Plex Media Server:** v1.3.3.3148
 - **Trakt.tv (for Plex):** v1.3.0

------------------

I would check the ownership and permissions on the `Plug-ins` directory itself, as that is what the error is actually referencing.

It should be:
```
drwxr-xr-x 3 plex nogroup 4096 Dec 30 18:46 Plug-ins
```

This can be set with:
```
chown plex:nogroup "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins"
chmod 755 "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins"
```

**Comment by brianmichaud** [comment]
Alright. I fixed it. For some reason everything for PMS was installed in /home/pi/Library/ instead of /var/lib/plexmediaserver . . .I put the trakttv.bundle in that other directory and it started up right away. Sorry for the confusion, and I sincerely appreciate the help. That would explain why the PMS logs I sent hadn't updated since 11/26, and why the other channels I had installed weren't displaying in the /var/lib/plexmediaserver directories. . . 

Thanks again!
